components/brave_rewards/resources/rewards_internals/components/contributions.tsx

```diff
@@ -0,0 +1,80 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import { ContributionInfo } from '../lib/app_state'
+import { useAppState, useAppActions } from '../lib/app_model_context'
+
+import { style } from './contributions.style'
+
+export function Contributions() {
+  const actions = useAppActions()
+  const contributions = useAppState((state) => state.contributions)
+
+  React.useEffect(() => { actions.loadContributions() }, [])
+
+  function renderContribution(contribution: ContributionInfo) {
+    return (
+      <div className='content-card'>
+        <h4>{contribution.id}</h4>
+        <section>
+          <div className='key-value-list'>
+            <div>
+              <span>Created at:</span>
```

Comment:
Are we explicitly dropping localization for rewards-internals?

Reply:
Good question. The page title and disclaimers are still translated, but I was unsure about whether localization was needed for labels and text on internals pages. It seems like most of our (and upstream's) internals pages are not localized, and much of the data itself presented on this page is not localized (all of the logs and ads diagnostics, for instance).

I think we should drop localization and remove the strings in a follow-up, unless there are issues.

Reply:
Yeah, that makes sense, just wanted to check.